platform/src/main/resources/anchor-config-defaults.yaml

```diff
@@ -1,29 +1,76 @@
+# This file configures the anchor platform server.
+
 stellar:
   anchor:
-    # Configure the application from this file after it is loaded in memory
-    # If `config` is in-memory, the yaml file contains all settings for the server.
+    # Set the source of the configuration. The configuration source determines how the anchor platofrm
+    # server fetch the configuration values.
+    #
+    # If `config` is `in-memory`, this yaml file contains all settings for the server.
+    #
+    # `in-memory` is the only `config` value supported in MVP.
+    #
     config: in-memory
+
+    #
+    # Sets how the application configuration values are read.
+    #
+    #  If `type` is `config-spring-property`, the platform server will use the Spring's `@ConfigruationProperties` to
+    #      read populate the configuration values.
+    #
+    #  `config-spring-property` is the only `type` value supported in MVP.
     app-config:
-      type: config-spring-property                # Activate [config-spring-property] module
-      settings: app-config                        # The location of the configuration data
+      type: config-spring-property
+      settings: app-config                        # The location of the configuration data in this yaml file
```

Comment:
Yes, we can rename one of them to something different. Since there is another PR depending on this PR, I had submitted [issue#243](https://github.com/stellar/java-stellar-anchor-sdk/issues/243).